### PR TITLE
Making installation/ramp-up easier.

### PR DESCRIPTION
**Pull request by korymath**
Great work on the project! I wanted to add my notes for getting this system running on its own. I find that it can help to make explicit lots of the steps of installation to help avoid issues in the future. 

This PR: 

- [x] includes explicit details on setting up a virtual environment for running MinAtar/
- [x] adds a `requirements.txt` file to address: https://github.com/kenjyoung/MinAtar/issues/1
- [x] adds a basic python .gitignore to ignore virtual environment and other temporary and unneeded python files.
- [x] adds a line in the README on expected output following proper installation.

Thank you again for releasing the code!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,28 +12,39 @@ MinAtar is a testbed for AI agents which implements miniaturized version of seve
...
 ## Quick Start
-MinAtar consists of a python3 package, to use MinAtar you must first install numpy. If you wish to visualize the game play (for example to play as a human) you must also install the packages seaborn and matplotlib. The included  DQN and AC_lambda examples are written using pytorch, and thus also require the torch package to run.
+MinAtar consists of a python3 package, to use MinAtar follow the installation instructions. The included `DQN` and `AC_lambda` examples are written using `PyTorch`.
 
-To install MinAtar simply do:
+To install MinAtar simply:
 
 ```bash
-pip3 install .
+git clone https://github.com/korymath/MinAtar.git
```
Comment (kenjyoung) [inline README.md]:
```suggestion
git clone https://github.com/kenjyoung/MinAtar.git
```

File: README.md [hunk 2]
```diff
@@ -12,28 +12,39 @@ MinAtar is a testbed for AI agents which implements miniaturized version of seve
...
+source venv/bin/activate
+# Upgrade Pip
+pip install --upgrade pip
+# Install requirements
+pip install -r requirements.txt
+# Install
+python setup.py install
+# Ensure installation success
+python examples/random_play.py -g breakout
+# It should run for a bit and then you should see an output:
```
Comment (kenjyoung) [inline README.md]:
```suggestion
# It should run for a bit and then you should see an output similar to:
```

File: README.md [hunk 3]
```diff
@@ -12,28 +12,39 @@ MinAtar is a testbed for AI agents which implements miniaturized version of seve
...
-MinAtar consists of a python3 package, to use MinAtar you must first install numpy. If you wish to visualize the game play (for example to play as a human) you must also install the packages seaborn and matplotlib. The included  DQN and AC_lambda examples are written using pytorch, and thus also require the torch package to run.
+MinAtar consists of a python3 package, to use MinAtar follow the installation instructions. The included `DQN` and `AC_lambda` examples are written using `PyTorch`.
 
-To install MinAtar simply do:
+To install MinAtar simply:
 
 ```bash
-pip3 install .
+git clone https://github.com/korymath/MinAtar.git
+# Configure a virtual environment
```
Comment (kenjyoung) [inline README.md]:
```suggestion
# Configure a virtual environment (optional)
```

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cycler==0.10.0
+kiwisolver==1.0.1
+matplotlib==3.0.3
+MinAtar==1.0.1
```
Comment (kenjyoung) [inline requirements.txt]:
```suggestion
```

**Comment by korymath** [comment]
Addressed your fixes, thank you for the quick turnaround.

**Pull request merged**